src/scalardl/src/main/java/com/example/contract/token/ExchangeToken.java

```diff
@@ -0,0 +1,62 @@
+package com.example.contract.token;
+
+import com.scalar.dl.ledger.asset.Asset;
+import com.scalar.dl.ledger.contract.Contract;
+import com.scalar.dl.ledger.database.Ledger;
+import com.scalar.dl.ledger.exception.ContractContextException;
+import java.util.Optional;
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+
+public class ExchangeToken extends Contract {
```

Comment:
Just a comment.
This contract looks almost the same as `SendToken` contract.
`ExchangeToken`, `IssueToken` and `SendToken` might be able to be implemented with the same contract.
However, this implementation looks simpler than that since these contracts are small.

Reply:
Thank you for the comment. Yeah, they are a bit redundant. I just wanted to keep them simple to show their functionalities.